Add requested configs + more + fixes

Whennya opened the pull request:
Adds configs for:
- ze_cyberspace_v1_s
- ze_dark_souls_ptd_v0_4
- ze_ffvii_malgo_reactor_css4_2
- ze_ffxii_mt_bur_omisace_v6fix
- ze_ffxii_salikawood_v1_8s
- ze_illya_b4
- ze_ouroboros_v3
- ze_rizomata_s2_1
- ze_luciddreams_css4
- ze_eerie_escape_redux_v2
- ze_ffxiv_wanderers_palace_v6css

Fixes for:
- ze_ffvii_malgo_reactor_css4_2
- ze_ffxiv_wanderers_palace_v5_2f
- ze_ffxiv_wanderers_palace_v6
- ze_lotr_isengard_v2_3
- ze_luciddreams_v3_6_f2
- ze_santassination_css2
- ze_xenovation_v3_5_s
- zs_obj_rescape_v19
- ze_ffxiv_wanderers_palace_v5_2f.cfg

Whennya committed:
Add unfinished config + run experimental script

Whennya committed:
Update ze_dark_souls_ptd_v0_4.cfg

Whennya committed:
run script

Whennya committed:
add configs

Added:
- ze_cyberspace_v1_s
- ze_esther_cc_zi2
- ze_luciddreams_css4
- ze_rizomata_s2

Whennya committed:
add

- ze_illya_b4
- ze_ouroboros_v3

Whennya committed:
update requested configs

Whennya committed:
remove end lines

Whennya committed:
oops

Whennya committed:
Update ze_rizomata_s2_1.cfg

Whennya committed:
add 2 more

- ze_eerie_escape_redux_v2
- ze_ffxiv_wanderers_palace_v6css

notkoen commented on musicname/ze_dark_souls_ptd_v0_4.cfg:
```diff
@@ -0,0 +1,23 @@
+"music"
+{
+    "anor_londo_music_1_1.mp3"          "Dark Souls 3 OST - Twin Princes"
+  //"anor_londo_music_2_1.mp3"          ""
```
According to legacy config, `anor_londo_music_2_1.mp3` should be `"Demon's Souls OST - Maiden Astraea (Hellkite Remix)"`. Will need to double check.

Whennya replied:
looks right, didnt check old config

notkoen commented on musicname/ze_eerie_escape_redux_v2.cfg:
```diff
@@ -0,0 +1,15 @@
+    "level5_end_song (1).mp3"       "Immediate Music - Dark Side Of Power"
+    "stage3_music2.mp3"             "Two Steps From Hell - Protectors of the Earth"
+    "stage4_music1.mp3"             "Pendulum - Crush"
+    "TSFH_Victory.mp3"              "Two Steps From Hell - Victory"
```
Capital file name

notkoen commented on musicname/ze_ffxii_salikawood_v1_8s.cfg:
Some of the music is from Kingdom (the anime?), follow `<Artist> - <Song Name> (Kingdom OST)` format in this case.

notkoen commented on musicname/ze_ffxii_salikawood_v1_8s.cfg:
```diff
@@ -0,0 +1,21 @@
+    "yamamoto kouta - hung in a bad place.mp3"                          "Hung in a Bad Place - Kingdom OST - KOHTA YAMAMOTO"
+    "studio eim - pvp.mp3"                                              "StudioEIM - Vindictus PVP Theme 3"
+    "studio eim - elchulus the ancient.mp3"                             "StudioEIM - 고대 엘쿨루스 (Elchulus the Ancient)"
+    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega(Epic Dark Orchestral)"
```
Remove `(Epic Dark Orchestral)` at the end, extra detail

notkoen replied:
Remove extra detail at the end
```suggestion
    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega"
```

notkoen commented on musicname/ze_ffxii_salikawood_v1_8s.cfg:
```diff
@@ -0,0 +1,21 @@
+    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega(Epic Dark Orchestral)"
+    "nishiki yasunori - octopath traveler.mp3"                          "Octopath Traveler OST - Main Theme"
+    "immediate music - atlas.mp3"                                       "Immediate Music - Atlas"
+    "hitoshi sakimoto - the dalmasca eastersand.mp3"                    "Final Fantasy XII OST - The Dalmasca Eastersand"
```
Follow conventions we agreed on from before; Use `FFXII OST`

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v6.cfg:
We can probably remove this file instead. `ze_ffxiv_wanderers_palace_v6` is a csgo version that wont be used. `v6css` is what's on sunrust so better change to this.

notkoen commented on musicname/ze_ffxiv_wanderers_palace_v5_2f.cfg:
If you don't mind, there are extra music on this version than the csgo version that you should check for.

notkoen commented on musicname/ze_ouroboros_v3.cfg:
```diff
@@ -0,0 +1,8 @@
+"music"
+{
+    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching CAvern without Lucidity"
```
Typo, capital A in `CAvern`
```suggestion
    "the caretaker - everywhere at the end of time - 35 f5 - aching cavern without lucidity.mp3"            "The Caretaker - Everywhere at The End of Time - 35 f5 - Aching Cavern without Lucidity"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
I see `NieR Replican ver.1.22` listed twice, one with and one without the OST. Also, for `5_music2_v2.mpe` the OST information should be in parentheses.
`5_bossmusic2.mp3` says `NieR Gestalt Replicant Original Soundtrack`, can be shortened to just `NieR Gestalt Replicant OST`

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "4_music4.mp3"                  "Two Steps From Hell - Clock Tower Parade"
+    "4_bossmusic_v2.mp3"            "NieR OST. - Emil Karma"
+    "5_music.mp3"                   "Fantasy Music - Age of Wonders"
+    "5_music2_v2.mp3"               "Kainé - Escape ｜ NieR Replicant ver 1.22 OST"
```
```suggestion
    "5_music2_v2.mp3"               "Kainé - Escape (NieR Replicant ver 1.22 OST)"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "1_music2.mp3"                  "Fantasy Medieval Music - Rise of a Kingdom"
+    "2_music.mp3"                   "Epic Fantasy Music - Mother Earth"
+    "2_music2_v2.mp3"               "Ori and the Blind Forest - Escaping the Ruins"
+    "2_bossmusic.mp3"               "NieR Replicant ver.1.22 - The Dark Colossus Destroys All (Percussion Mix)"
```
```suggestion
    "2_bossmusic.mp3"               "NieR Replicant ver 1.22 - The Dark Colossus Destroys All"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "2_music2_v2.mp3"               "Ori and the Blind Forest - Escaping the Ruins"
+    "2_bossmusic.mp3"               "NieR Replicant ver.1.22 - The Dark Colossus Destroys All (Percussion Mix)"
+    "3_music.mp3"                   "Ivan Torrent - Human Legacy"
+    "3_music2.mp3"                  "Ramin Djawadi - Main Title[Game of Thrones]"
```
```suggestion
    "3_music2.mp3"                  "Ramin Djawadi - Main Title (Game of Thrones OST)"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "2_bossmusic.mp3"               "NieR Replicant ver.1.22 - The Dark Colossus Destroys All (Percussion Mix)"
+    "3_music.mp3"                   "Ivan Torrent - Human Legacy"
+    "3_music2.mp3"                  "Ramin Djawadi - Main Title[Game of Thrones]"
+    "3_music3.mp3"                  "Mhysa - Game of Thrones - Season 3 - Soundtrack"
```
```suggestion
    "3_music3.mp3"                  "Game of Thrones: Season 3 OST - Mhysa"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "3_music2.mp3"                  "Ramin Djawadi - Main Title[Game of Thrones]"
+    "3_music3.mp3"                  "Mhysa - Game of Thrones - Season 3 - Soundtrack"
+    "3_bossmusic_v2.mp3"            "NieR OST - Gods Bound by Rules"
+    "3_music4.mp3"                  "Finale- Game of Thrones - Season 1 - Soundtrack"
+    "4_music.mp3"                   "Two Steps From Hell - El Dorado (SkyWorld)"
```
Same stuff but I think you get it

Whennya committed:
comments

Whennya committed:
run script

Whennya committed:
Update ze_ffxii_salikawood_v1_8s.cfg

notkoen commented on musicname/ze_ffxii_salikawood_v1_8s.cfg:
```diff
@@ -0,0 +1,21 @@
+    "nishiki yasunori - octopath traveler.mp3"                          "Octopath Traveler OST - Main Theme"
+    "immediate music - atlas.mp3"                                       "Immediate Music - Atlas"
+    "hitoshi sakimoto - the dalmasca eastersand.mp3"                    "FFXII OST - The Dalmasca Eastersand"
+    "hitoshi sakimoto - streets of rabanastre.mp3"                      "FFXII: The Zodiac Age OST Streets of Rabanastre"
```
```suggestion
    "hitoshi sakimoto - streets of rabanastre.mp3"                      "FFXII: The Zodiac Age OST - Streets of Rabanastre"
```

notkoen commented on musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg:
```diff
@@ -32,10 +32,17 @@
+    "audiomachine_smoke_screen.mp3"                             "Audiomachine - Smoke Screen"
+    "metrik_time_to_let_go.mp3"                                 "Metrik - Time To Let Go"
+    "phoenix_rising2.mp3"                                       "Edgar Hopp - Phoenix Rising"
+    "zardonic_follow_me2.mp3"                                   "Zardonic - Follow Me"
```
```suggestion
    "zardonic_follow_me2.mp3"                                   "Zardonic - Follow Me (ft. Camo MC)"
```

notkoen commented on musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg:
```diff
@@ -32,10 +32,17 @@
+    "metrik_time_to_let_go.mp3"                                 "Metrik - Time To Let Go"
+    "phoenix_rising2.mp3"                                       "Edgar Hopp - Phoenix Rising"
+    "zardonic_follow_me2.mp3"                                   "Zardonic - Follow Me"
+    "zardonic_takeover.mp3"                                     "Zardonic - Takeover"
```
```suggestion
    "zardonic_takeover.mp3"                                     "Zardonic - Takeover (ft. The Qemists)"
```

notkoen commented on musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg:
```diff
@@ -32,10 +32,17 @@
     // Other
     "pendulum_witchcraft.mp3"                                   "Pendulum - Witchcraft (Live at Glostonbury 2011)"
     "pendulum_ghosts_n_stuff.mp3"                               "deadmau5 ft. Rob Swire - Ghosts 'n Stuff (Pendulum Remix)"
     "sinners_heist_streetlight_people.mp3"                      "Sinners Heist ft. Harley - Streetlight People"
```
```suggestion
    "sinners_heist_streetlight_people.mp3"                      "Sinners Heist - Streetlight People (ft. Harley)"
```

notkoen commented on musicname/ze_ffxii_mt_bur_omisace_v6fix.cfg:
`the_twelve_towers.mp3` and `voicians_uncharted.mp3` are deleted from the map based on what Malgo said.

Whennya committed:
updates

updates:
- ze_ffxiv_wanderers_palace_v5_2f.
- ze_ffxii_salikawood_v1_8s
- ze_ffxii_mt_bur_omisace_v6fix

Whennya committed:
Update ze_ffxiv_wanderers_palace_v5_2f.cfg

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "4_music4.mp3"                  "Two Steps From Hell - Clock Tower Parade"
+    "4_bossmusic_v2.mp3"            "NieR Replicant ver.1.22 OST - Emil Karma"
+    "5_music.mp3"                   "Fantasy Music - Age of Wonders"
+    "5_music2_v2.mp3"               "NieR Replicant ver 1.22 OST - Kainé / Escape"
```
```suggestion
    "5_music2_v2.mp3"               "NieR Replicant ver.1.22 OST - Kainé / Escape"
```

notkoen commented on musicname/ze_rizomata_s2_1.cfg:
```diff
@@ -0,0 +1,22 @@
+    "4_bossmusic_v2.mp3"            "NieR Replicant ver.1.22 OST - Emil Karma"
+    "5_music.mp3"                   "Fantasy Music - Age of Wonders"
+    "5_music2_v2.mp3"               "NieR Replicant ver 1.22 OST - Kainé / Escape"
+    "5_bossmusic2.mp3"              "NieR Gestalt Replicant Original Soundtrack. Deep Crimson Foe"
```
```suggestion
    "5_bossmusic2.mp3"              "NieR Gestalt Replicant OST - Deep Crimson Foe"
```

Whennya committed:
Update ze_rizomata_s2_1.cfg
